.templates/template_location.json

```diff
@@ -3,14 +3,26 @@
     "repoPath": "https://github.com/GoogleCloudPlatform/cloud-code-samples.git",
     "directoryPath": "python/python-hello-world",
     "templatePath": "python-hello-world",
-    "templateName": "Python: Hello World"
+    "templateName": "Python (Flask): Hello World"
```

Comment:
Could you please rebase your local branch so template_location.json is not part of your change?
I wanted to submit this change separately (addition of django)

Reply:
Oops. Didnt catch that. Will update.

Reply:
fixed. merged from upstream